tests/support/serversupp.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# --- BEGIN_HEADER ---
+#
```

Comment:
nit-pick: one-liner with module name and description should always go here. Not sure you saw my comment in previous review, and I'll just add it in merge.

Reply:
Yeah - I guess I have kept trying to trigger some traction on revisiting what's in these headers but we see very keen to continue with what we have - since it's only serving to generate otherwise unnecessary review comments I think this is squarely in bigger fish to fry territory.